<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MakeSealWindow</class>
 <widget class="QDialog" name="MakeSealWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>771</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>印章制作</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>签发人/单位</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="company">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>签章名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>有效时间至</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>310</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwd">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>310</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>380</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>图片</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>380</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="icon">
    <iconset resource="../../图标资源.qrc">
     <normaloff>:/resource/图片/open.png</normaloff>:/resource/图片/open.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>560</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>制作</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>630</y>
     <width>401</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>证书</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="certificate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getCertificateButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>10</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../图标资源.qrc">
     <normaloff>:/resource/图片/open.png</normaloff>:/resource/图片/open.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>440</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>印章生成位置</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sealDialog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>440</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../图标资源.qrc">
     <normaloff>:/resource/图片/open.png</normaloff>:/resource/图片/open.png</iconset>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="endtime">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pictrue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>301</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>证书私钥</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="certificateKey">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>301</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="keyButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../图标资源.qrc">
     <normaloff>:/resource/图片/open.png</normaloff>:/resource/图片/open.png</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>490</y>
     <width>68</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>印章类型</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sealType">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>个人印章</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>企业印章</string>
    </property>
   </item>
  </widget>
  <zorder>endtime</zorder>
  <zorder>label_2</zorder>
  <zorder>company</zorder>
  <zorder>label_3</zorder>
  <zorder>name</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>pwd</zorder>
  <zorder>label_6</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>pushButton</zorder>
  <zorder>textBrowser</zorder>
  <zorder>label_7</zorder>
  <zorder>certificate</zorder>
  <zorder>getCertificateButton</zorder>
  <zorder>label_8</zorder>
  <zorder>sealDialog</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pictrue</zorder>
  <zorder>label_12</zorder>
  <zorder>certificateKey</zorder>
  <zorder>keyButton</zorder>
  <zorder>label</zorder>
  <zorder>sealType</zorder>
 </widget>
 <tabstops>
  <tabstop>certificate</tabstop>
  <tabstop>getCertificateButton</tabstop>
  <tabstop>certificateKey</tabstop>
  <tabstop>keyButton</tabstop>
  <tabstop>company</tabstop>
  <tabstop>name</tabstop>
  <tabstop>endtime</tabstop>
  <tabstop>pwd</tabstop>
  <tabstop>pictrue</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>sealDialog</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources>
  <include location="../../图标资源.qrc"/>
 </resources>
 <connections/>
</ui>
